include/vrv/svgdevicecontext.h

```diff
@@ -225,6 +225,7 @@ class SvgDeviceContext : public DeviceContext {
             std::string className = s.substr(0, s.find("@")); // parse <element@attribute>, e.g., "note@pname"
             std::string attributeName = s.substr(s.find("@") + 1);
             ClassId classId = ObjectFactory::GetInstance()->GetClassId(className);
+            assert(classId);
```

Comment:
Do we have to check whether the classId (or the attribute string) exists before inserting it to the multimap?

Reply:
No, you don't want an `assert` here because this is going to make Verovio crash if the user passes a parameter with a typo in the element name or an element name that is not supported by Verovio. I think what you can do is simply ignore it (i.e., not add it to the map) if `(classId == OBJECT)`

Reply:
Ok, great. Yes, wrong syntax is simply ignored now. Or, an error is thrown when the ClassId is not correct.